docs/LV_client/tutorials/wheel_integration.md

```diff
@@ -0,0 +1,31 @@
+# Wheel integration
```

Comment:
Capitalize Integration

Reply:
Speaking of, would it be possible to go ahead and capitalize all the tab titles like this?